src/components/FileAnnotation/FileEditor.js

```diff
@@ -116,11 +119,13 @@ FileEditor.propTypes = {
   /** The value of the currently selected fileType's enum*/
   fileType: PropTypes.string,
   /** Action to perform when form is submitted */
-  onSubmit: PropTypes.func.isRequired,
+  onSubmit: PropTypes.func,
   /** Action to perform when name input is updated */
   onNameChange: PropTypes.func.isRequired,
   /** Function for onChange event on file type selection */
   selectFileType: PropTypes.func.isRequired,
+  /** If to hide the save/cancel buttons and badge */
+  showStatus: PropTypes.bool,
```

Comment:
@XuTheBunny I can't remember if we removed this in favor of `renderButtons`

Reply:
`showStatus` is in use to conditionally render edit/save button near file name input and the badge underneath the file name input